src/components/Toolbar.ts

```diff
@@ -74,6 +81,52 @@ const toolbarView = (vnode: Vnode<IToolbar>) => {
           },
           "Import HAR Session"
         ),
+        m(
+          "a.button.is-outlined.is-small.control",
+          {
+            class: ngrokStreamLock === -1 ? "ngrok-play" : "ngrok-stop"
+          },
+          m("hi", {
+            onclick: () => {
+            if (ngrokStreamLock != -1) {
+              console.log('stopping ngrok stream');
+              ngrokStreamLock = -1;
+              return;
+            }
+
+            // this permissions things works but it needs a better way
+            // to test and reset it!
```

Comment:
Yeah I haven't figured that out properly either.

I think if you go into the Manage Add-ons thing into the details it has a list of granted permissions which you can clear/remove stuff from. It should then ask you again.